client/packages/common/src/ui/medical-icons/Contraceptive.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const ContraceptiveIcon = (props: SvgIconProps): JSX.Element => (
+  <SvgIcon {...props} viewBox="0 0 60 60" fill="none">
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M30.5025 10L12 14.5644V24.0421C12 36.6282 19.6174 47.6823 30.1059 50.3278L30.5102 50.4289L30.8909 50.3278C41.3826 47.6792 49 36.625 49 24.0388V14.564L30.5025 10ZM47.0308 24.0421C47.0308 35.6978 40.0926 45.9195 30.4977 48.3976C20.9108 45.9274 13.9646 35.7027 13.9646 24.0421V16.1057L30.5005 12.0261L47.0304 16.1057L47.0308 24.0421ZM15.7318 24.0421V17.4925L30.5025 13.8457L45.2635 17.4925V24.0421C45.2635 34.7251 39.0846 44.0971 30.4977 46.5672C21.9058 44.0924 15.7318 34.7251 15.7318 24.0421ZM30.6595 37.9892C35.3828 37.9892 39.2117 34.1602 39.2117 29.437C39.2117 24.7137 35.3828 20.8847 30.6595 20.8847C25.9362 20.8847 22.1072 24.7137 22.1072 29.437C22.1072 34.1602 25.9362 37.9892 30.6595 37.9892ZM30.6139 27.0912C30.9007 26.4423 31.6943 25.9383 32.63 25.9383C33.7828 25.9383 35.0053 26.5871 35.1501 28.4556V28.6004C35.1501 31.9142 30.6139 33.7131 30.6139 33.7131C30.6139 33.7131 26.1501 31.9114 26.1501 28.6004V28.3136C26.2949 26.5871 27.4449 25.9383 28.5978 25.9383C29.5334 25.9383 30.3271 26.37 30.6139 27.0912Z"
+      fill="url(#paint0_linear_247_41468)"
+    />
+    <defs>
+      <linearGradient
+        id="paint0_linear_247_41468"
+        x1="30.6501"
+        y1="25.9383"
+        x2="30.6501"
+        y2="33.7131"
+        gradientUnits="userSpaceOnUse"
+      >
+        <stop stop-color="#E73735" />
```

Comment:
If moving to components, please use the theme rather than hard coded colours.
These are also colours outside of our current theme which isn't ideal. Perhaps should be primary.main and primary.light?

Just use `useTheme` to pull in the theme and take the colours as needed.